internal/controllers/clusterextension_controller.go

```diff
@@ -135,25 +135,16 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 
 	// TODO: Question - Should we set the deprecation statuses after we have successfully resolved instead of after a successful installation?
 
-	mediaType, err := bundle.MediaType()
-	if err != nil {
-		setInstalledStatusConditionFailed(&ext.Status.Conditions, err.Error(), ext.GetGeneration())
-		setDeprecationStatusesUnknown(&ext.Status.Conditions, "deprecation checks have not been attempted as installation has failed", ext.GetGeneration())
-		return ctrl.Result{}, err
-	}
-
 	if err := r.validateBundle(bundle); err != nil {
 		setInstalledStatusConditionFailed(&ext.Status.Conditions, err.Error(), ext.GetGeneration())
 		setDeprecationStatusesUnknown(&ext.Status.Conditions, "deprecation checks have not been attempted as installation has failed", ext.GetGeneration())
 		return ctrl.Result{}, err
 	}
 
-	bundleProvisioner, err := mapBundleMediaTypeToBundleProvisioner(mediaType)
-	if err != nil {
-		setInstalledStatusConditionFailed(&ext.Status.Conditions, err.Error(), ext.GetGeneration())
-		setDeprecationStatusesUnknown(&ext.Status.Conditions, "deprecation checks have not been attempted as installation has failed", ext.GetGeneration())
-		return ctrl.Result{}, err
-	}
+	// Assume all bundles are registry+v1 for now, since that's all we support.
+	// If the bundle is not a registry+v1 bundle, the conversion will fail.
+	bundleProvisioner := "core-rukpak-io-registry"
```

Comment:
Nit: Maybe let's get rid of `bundleProvisioner` variable and of param in `r.GenerateExpectedBundleDeployment`?

Will help to reduce the noise in `reconcile` a little bit.